### PR TITLE
feat: [CDS-80203]: Settings page UX fixes and added icons for tiles

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
### Summary

Added Icons and small settings page css fix.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
trigger messagecheck

**Comment by Sagar-Rajput7** [comment]
trigger messagecheck

**Review by rameshx — APPROVED** [review]

**Review by MondalSayantan — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Review by srdjan-harness — APPROVED** [review]

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/10-common/pages/SettingsPages/AccountSettingsPage.tsx [hunk 1]
```diff
@@ -398,7 +398,7 @@ export const AccountSettingsPage: React.FC = () => {
             <SettingsResourceCard
               label={<String stringID="common.freezeWindows" />}
               id={SettingsResources.FreezeWindow}
-              icon={'setting'}
+              icon={'FreezeWindow'}
```
Comment (chetanvekariya-harness) [inline src/modules/10-common/pages/SettingsPages/AccountSettingsPage.tsx:401]: icons names are generally in lower case separated by `-`. We could have maintained consistency/

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by namanharness — APPROVED** [review]

File: src/modules/10-common/pages/SettingsPages/AccountSettingsPage.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/10-common/pages/SettingsPages/AccountSettingsPage.tsx:401]: This was an existing icon, I will change according to the standards in the coming pr.

**Pull request merged**